java/gradle/ice.gradle

```diff
@@ -52,6 +54,15 @@ slice {
     }
 }
 
+spotless {
+  java {
+    target project.fileTree("src/main/java") {
```

Comment:
Should we include the tests as well?

Reply:
This is the project `src/main/java` directory. All projects including tests have this structure.